### PR TITLE
edits to make docs work both in new ehcache.org site (with jekyll) an…

### PR DESCRIPTION
**Pull request by jhouserizer**
edits to make docs work both in new ehcache.org site (with jekyll) and as stand-alone (somewhat broken now though)

### CODE REVIEW

**Comment by jhouserizer** [comment]
does not seem that that build failure is related to the changes.

**Comment by ljacomet** [comment]
test this please

**Comment by jhouserizer** [comment]
test this please

File: docs/src/docs/asciidoc/user/menu.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
- link:../../../index.html[Home]::
+link:../../../index.html[Home]::
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/menu.adoc]: Needs to be changed to simple `index.html` without the navigation

File: docs/src/docs/asciidoc/user/menu.adoc [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
- - link:./index{outfilesuffix}[Getting started]
- - link:./xml{outfilesuffix}[XML Configuration]
- - link:./usermanaged{outfilesuffix}[User managed caches]
- - link:./107{outfilesuffix}[javax.cache aka jsr-107]
- - link:./management{outfilesuffix}[Management and Monitoring]
- Resources::
- - link:./examples{outfilesuffix}[Java examples]
- - link:./xsds{outfilesuffix}[Configuration XSD]
+User documentation::
+- link:./index{outfilesuffix}[Getting started]
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/menu.adoc]: Needs `index` to be replaced with `getting-started`

File: docs/src/docs/asciidoc/user/menu.adoc (hunk 1 above)
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/menu.adoc]: And the comment below removed as I believe it no longer applies

**Pull request merged**